dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtility.java

```diff
@@ -590,6 +591,9 @@ private List<File> downloadFeatureEsas(Collection<String> resolvedFeatures) thro
         if (map.get("action.error.message") != null) {
```

Comment:
Can we use constants for `action.error.message` and `action.error.stacktrace`?

Reply:
I will create a seperate PR for this